test/trento/infrastructure/commanded/roll_up/roll_up_test.exs

```diff
@@ -1,7 +1,7 @@
-defmodule Trento.RollUpTest do
+defmodule Trento.Infrastructure.Commanded.RollUp.RollUpTest do
```

Comment:
Shouldn't this be `Commanded.RollUpTest` simply, unless you change the lib code

Reply:
The lib code is `Trento.Infrastructure.Commanded.RollUp`

Reply:
yep, thx